search-service/src/main/kotlin/com/egm/stellio/search/util/AttributeInstanceUtils.kt

```diff
@@ -15,3 +19,17 @@ fun valueToStringOrNull(value: Any): String? =
         is String -> value
         else -> null
     }
+
+fun buildAttributeInstancePayload(
+    value: Any,
+    observedAt: ZonedDateTime,
+    datasetId: URI? = null,
+    instanceId: URI? = null
+): String {
+    val payload = mutableMapOf<String, Any>("type" to "Property")
+    datasetId?.let { payload["datasetId"] = it }
+    payload["value"] = value
+    instanceId?.let { payload["instanceId"] = it }
+    payload["observedAt"] = observedAt
```

Comment:
can't this be improved?

Reply:
The reason behind this is that attributes are expected in a certain order.
It can be improved but we should also update the expected data.

Reply:
they are expected *in the tests*. as you know, you don't hack the implementation to please your tests.

Reply:
you are right, i'll fix it.

Reply:
only used in tests, I've moved this function.